test/e2e/utils/request.js

```diff
@@ -0,0 +1,102 @@
+
+const axios = require('axios');
+const config = require('./config/config');
+const auth = require('./authUtil');
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+axios.defaults.headers.common['Content-Type'] = 'application/json';
+const axiosOptions = {
+    baseURL: config.getBaseUrl()
+};
+axios.defaults.withCredentials = true;
+axios.defaults.baseURL = config.getBaseUrl();
+const http = axios.create(axiosOptions);
+
+const requestInterceptor = (request) => {
+    console.log("Base url:::::::"+config.getBaseUrl());
+    console.log("requestInterceptor:::::::"+`${request.method.toUpperCase()} to ${request.url}`);
+    return request;
+};
+http.interceptors.request.use(requestInterceptor);
+
+
+class Request {
+
+     cookieString = '';
+     async withSession(username, password) {
+        this.cookieString = await auth.getSessionCookieString(username, password);
```

Comment:
instead of login and getting cookies from another module, we can reuse the cookies from existing protractor browser session to keep test data in sync with logged in user in browser. 
set cookie string value from cookies in protractor browser to resolve this .